portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,144 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
+const COMMENT_SIZE_LIMIT = 100;
+
 function getRandomFact() {
-  const greetings =
-      ['My ex podmate stole my go link', 'I was the shortest person in my junior high', 'I know 3 languages', 'C++ is my favorite programming language'];
+  const facts =
+    ["My ex podmate stole my go link", "I was the shortest person in my junior high", "I know 3 languages", "C++ is my favorite programming language"];
```

Comment:
Consider pulling this array outside of the function so that it is only initialized once when the application first loads.